test_integration_app/psp_merchant_api/test_psp_merchant_api.php

```diff
@@ -7,11 +7,12 @@ class test_psp_merchant_api
         function __construct() {
             $this->client = new SiftClient(array('api_key' => getenv("api_key"), 'account_id' => getenv("account_id")));
         }
-        
+
         function create_merchant()
         {
+            $merchant_id = rand(1, 1000000); 
             $merchantObject = array(
-                'id' => "merchant-id-php-0009",
+                'id' => "merchant-id-php-".$merchant_id,
```

Comment:
btw, please check if we have the same problem in sift-python

Reply:
I have already added this logic in python